extensions/typescript/src/features/taskProvider.ts

```diff
@@ -0,0 +1,110 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Thanks. I've updated the activation as you described and it seems to work well. Will take a look at scanning for `tsconfig.json` outside of the root in a follow up PR